### DISCUSSION TITLE
[Arcropolis 3.7.0] Cannot connect to Smash Online

### DISCUSSION THREAD

**Discussion by EanWithHam** [Raytwo/ARCropolis#417] [Issues] [answered]
System Firmware Version: 14.1.2

Atmosphere Version: 1.3.2

Smash Version: 13.0.1

ARCropolis Version: 3.7.0

Can the error be consistently reproduced?: Yes

I was just playing online earlier today on Arcropolis 3.6.2, came back just now, launched the game, and accepted the prompt to download 3.7.0, and now I can no longer use the Online option on the Main Menu. It attempts to connect, then gives "Error Code 2306-0502: A server communication error has occurred. Please try again later." Checking the details option of this will have it check the server status, then say that "The server is operation normally. Please try again later." 
When I went to the Smash Ultimate Modding Hub Discord Server, I found that someone else also seems to be having the same issue.

**Comment by Raytwo** [comment, marked as answer]
You're right, that's on me, I'll push a fix soon

**Reply by Raytwo** [reply]
Fixed

**Reply by EanWithHam** [reply]
big thanks :)
